declarations/Qwen Chat.json

```
@@ -5,7 +5,7 @@
       "fetch": "https://chat.qwen.ai/legal-agreement/terms-of-service",
       "executeClientScripts": true,
       "select": [
-        ".legalPage"
+        ".app"
```

Will work, but might be more stable with:

```suggestion
        ".terms-of-service-legal-page"
```